tap-snapshots/test-cli.js-TAP.test.js

```diff
@@ -0,0 +1,11 @@
+/* IMPORTANT
```

Comment:
what is this :o

Reply:
http://www.node-tap.org/snapshots/

Reply:
I added a comment in the test about how to generate the file